crates/swc/tests/fixture/sourcemap/004/output/index.map

```diff
@@ -1,38 +1,33 @@
 {
-  "mappings": "AEACA,CAAAA,KAAK,gBAAmB,GAAGA,KAAK,gBAAmB,IAAI,EAAE,AAAD,EAAGC,IAAI,CAAC;IAC7D;QAAC;KAAI;IACL;QACU,MAAY,SACdC,CAAuB,EACvBC,CAAmB,EACnBC,CAAmB,EACrB;YACE;gBFNXC,IAAA,SAAAC,CAAA,EAAA;gBAAA,IAAAC,IAAAD,EAAAC,IAAA;gBAAA,OAAA,CAAA,GCDDC,EAAAC,GAAA,EAAA,OAAA;oBACAC,UAAAH,EAAAI,GAAA;gBACA;YACA;YAKYP,EAAoBQ,CAAC,CAACT,IACDC,EAAoBS,CAAC,CAACV,GAAqB;gBACvCW,SAAS,WAAY;oBACtC,OAAqBA;gBDZjB;gBAAqBC,SAAA,WAAA;oBAC3C,OAAOV;gBAAM;YACd;YAAA,IAAAG,IAAAJ,EAAA,OAAAU,IAAA,CAAA;QCKM;QAKP,MAAA,SAAAZ,CAAA,EAAAc,CAAA,EAAAZ,CAAA,EAAA;YAAAa,CAAAA,OAAAC,QAAA,GAAAD,OAAAC,QAAA,IAAA,EAAA,EAAAjB,IAAA,CAAA;gBAAA;gBAAA,WAAA;oBAAA,OAAAG,EAAA;gBAAA;aAAA;QAAA;IAAA;IAAA,SAAAA,CAAA,EAAA;QAAAA,EAAAe,CAAA,CAAA,GAAA;YAAA;YAAA;YAAA;SAAA,EAAA,WAAA;YAAA,OAAAf,EAAAA,EAAAgB,CAAA,GAAA;QAAA,IAAAC,OAAAjB,EAAAe,CAAA;IAAA;CAAA",
+  "mappings": "MACIC,gBAAA,GAAAD,KAAAC,gBAAA,IAAA,EAAA,EAAAC,IAAA,CAAA;;QAEC;KACD;;cAEQ,SAAAC,CAAA,EAAAC,CAAA,EAAAC,CAAA,EAAA;YACA;gBACIC,IAAI,SAAYH,CAAI,EAAA;gBACpB,IAAAC,IAAsBD,EAAAI,IAAkD;uBACpE,CAAA,GAAAC,EAAAC,GAAe,AAAL,EAAK,OAAG;oBACtBC,UAAAN,EAAAO,GAAA;gBACJ;YACA;gBAEyBP,IAAAC,EAAAO,CAAA,CAAAR,GAAS;yBAC1B,WAAqB;oBACzB,OAAAU;gBACqB;yBACjB,WAAqB;oBACzB,OAAAR;gBACJ;YACqB,EAAA;YAGjC,IAAAE,IAAAH,EAAA,OAAAS,IAAA,CAAA;QACc;cACD,SAAOX,CAAA,EAAAC,CAAA,EAAQC,CAAA,EAAG;oBACfY,QAAA,GAAAD,OAAAC,QAAA,IAAA,EAAA,EAAAf,IAAA,CAAA;gBACA;2BACW;oBACX,OAAAG,EAAA;gBACH;aAGb;QACI;IACS;aAK0BF,CAAA,EAAG;YAC9B,GAAA;YACA;YACA;YACD;sBAN4B;YAUtB,OAD0BA,EAAoBA,EAACgB,CAAA,GAAA;QAGhE,IAAAC,OAAAjB,EAAAe,CAAA,EAAA;IACC",
```

Comment:
I'll fix this with another PR
(Seems like a source map issue related to codegen of multiline comments.)

Reply:
[Visualized](https://evanw.github.io/source-map-visualization/#MTAxNQAoc2VsZi53ZWJwYWNrQ2h1bmtfTl9FID0gc2VsZi53ZWJwYWNrQ2h1bmtfTl9FIHx8IFtdKS5wdXNoKFsKICAgIFsKICAgICAgICAxNTgKICAgIF0sCiAgICB7CiAgICAgICAgMjk0MzogZnVuY3Rpb24obiwgdCwgdSkgewogICAgICAgICAgICAidXNlIHN0cmljdCI7CiAgICAgICAgICAgIHZhciByID0gZnVuY3Rpb24obikgewogICAgICAgICAgICAgICAgdmFyIHQgPSBuLmRhdGE7CiAgICAgICAgICAgICAgICByZXR1cm4gKDAsIF8uanN4KSgiZGl2IiwgewogICAgICAgICAgICAgICAgICAgIGNoaWxkcmVuOiB0LmZvbwogICAgICAgICAgICAgICAgfSk7CiAgICAgICAgICAgIH07CiAgICAgICAgICAgIHUucih0KSwgdS5kKHQsIHsKICAgICAgICAgICAgICAgIF9fTl9TU0c6IGZ1bmN0aW9uKCkgewogICAgICAgICAgICAgICAgICAgIHJldHVybiBpOwogICAgICAgICAgICAgICAgfSwKICAgICAgICAgICAgICAgIGRlZmF1bHQ6IGZ1bmN0aW9uKCkgewogICAgICAgICAgICAgICAgICAgIHJldHVybiByOwogICAgICAgICAgICAgICAgfQogICAgICAgICAgICB9KTsKICAgICAgICAgICAgdmFyIF8gPSB1KDQ1MTIpLCBpID0gITA7CiAgICAgICAgfSwKICAgICAgICA3MTM5OiBmdW5jdGlvbihuLCB0LCB1KSB7CiAgICAgICAgICAgICh3aW5kb3cuX19ORVhUX1AgPSB3aW5kb3cuX19ORVhUX1AgfHwgW10pLnB1c2goWwogICAgICAgICAgICAgICAgIi9zdGF0aWMiLAogICAgICAgICAgICAgICAgZnVuY3Rpb24oKSB7CiAgICAgICAgICAgICAgICAgICAgcmV0dXJuIHUoMjk0Myk7CiAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgIF0pOwogICAgICAgIH0KICAgIH0sCiAgICBmdW5jdGlvbihuKSB7CiAgICAgICAgbi5PKDAsIFsKICAgICAgICAgICAgNzc0LAogICAgICAgICAgICA4ODgsCiAgICAgICAgICAgIDE3OQogICAgICAgIF0sIGZ1bmN0aW9uKCkgewogICAgICAgICAgICByZXR1cm4gbihuLnMgPSA3MTM5KTsKICAgICAgICB9KSwgX05fRSA9IG4uTygpOwogICAgfQpdKTsKMjEzNQB7CiAgIm1hcHBpbmdzIjogIk1BQ0lDLGdCQUFBLEdBQUFELEtBQUFDLGdCQUFBLElBQUEsRUFBQSxFQUFBQyxJQUFBLENBQUE7O1FBRUM7S0FDRDs7Y0FFUSxTQUFBQyxDQUFBLEVBQUFDLENBQUEsRUFBQUMsQ0FBQSxFQUFBO1lBQ0E7Z0JBQ0lDLElBQUksU0FBWUgsQ0FBSSxFQUFBO2dCQUNwQixJQUFBQyxJQUFzQkQsRUFBQUksSUFBa0Q7dUJBQ3BFLENBQUEsR0FBQUMsRUFBQUMsR0FBZSxBQUFMLEVBQUssT0FBRztvQkFDdEJDLFVBQUFOLEVBQUFPLEdBQUE7Z0JBQ0o7WUFDQTtnQkFFeUJQLElBQUFDLEVBQUFPLENBQUEsQ0FBQVIsR0FBUzt5QkFDMUIsV0FBcUI7b0JBQ3pCLE9BQUFVO2dCQUNxQjt5QkFDakIsV0FBcUI7b0JBQ3pCLE9BQUFSO2dCQUNKO1lBQ3FCLEVBQUE7WUFHakMsSUFBQUUsSUFBQUgsRUFBQSxPQUFBUyxJQUFBLENBQUE7UUFDYztjQUNELFNBQU9YLENBQUEsRUFBQUMsQ0FBQSxFQUFRQyxDQUFBLEVBQUc7b0JBQ2ZZLFFBQUEsR0FBQUQsT0FBQUMsUUFBQSxJQUFBLEVBQUEsRUFBQWYsSUFBQSxDQUFBO2dCQUNBOzJCQUNXO29CQUNYLE9BQUFHLEVBQUE7Z0JBQ0g7YUFHYjtRQUNJO0lBQ1M7YUFLMEJGLENBQUEsRUFBRztZQUM5QixHQUFBO1lBQ0E7WUFDQTtZQUNEO3NCQU40QjtZQVV0QixPQUQwQkEsRUFBb0JBLEVBQUNnQixDQUFBLEdBQUE7UUFHaEUsSUFBQUMsT0FBQWpCLEVBQUFlLENBQUEsRUFBQTtJQUNDIiwKICAibmFtZXMiOiBbCiAgICAic2VsZiIsCiAgICAid2VicGFja0NodW5rX05fRSIsCiAgICAicHVzaCIsCiAgICAibiIsCiAgICAidCIsCiAgICAidSIsCiAgICAiciIsCiAgICAiZGF0YSIsCiAgICAiXyIsCiAgICAianN4IiwKICAgICJjaGlsZHJlbiIsCiAgICAiZm9vIiwKICAgICJkIiwKICAgICJfX05fU1NHIiwKICAgICJpIiwKICAgICJkZWZhdWx0IiwKICAgICJ3aW5kb3ciLAogICAgIl9fTkVYVF9QIiwKICAgICJPIiwKICAgICJzIiwKICAgICJfTl9FIgogIF0sCiAgInNvdXJjZXMiOiBbCiAgICAiLi4vLi4vaW5wdXQvaW5kZXguanMiCiAgXSwKICAic291cmNlc0NvbnRlbnQiOiBbCiAgICAiKHNlbGYud2VicGFja0NodW5rX05fRSA9IHNlbGYud2VicGFja0NodW5rX05fRSB8fCBbXSkucHVzaChbXG4gICAgW1xuICAgICAgICAxNThcbiAgICBdLFxuICAgIHtcbiAgICAgICAgMjk0MzogZnVuY3Rpb24obiwgdCwgdSkge1xuICAgICAgICAgICAgXCJ1c2Ugc3RyaWN0XCI7XG4gICAgICAgICAgICB2YXIgciA9IGZ1bmN0aW9uKG4pIHtcbiAgICAgICAgICAgICAgICB2YXIgdCA9IG4uZGF0YTtcbiAgICAgICAgICAgICAgICByZXR1cm4gKDAsIF8uanN4KShcImRpdlwiLCB7XG4gICAgICAgICAgICAgICAgICAgIGNoaWxkcmVuOiB0LmZvb1xuICAgICAgICAgICAgICAgIH0pO1xuICAgICAgICAgICAgfTtcbiAgICAgICAgICAgIHUucih0KSwgdS5kKHQsIHtcbiAgICAgICAgICAgICAgICBfX05fU1NHOiBmdW5jdGlvbigpIHtcbiAgICAgICAgICAgICAgICAgICAgcmV0dXJuIGk7XG4gICAgICAgICAgICAgICAgfSxcbiAgICAgICAgICAgICAgICBkZWZhdWx0OiBmdW5jdGlvbigpIHtcbiAgICAgICAgICAgICAgICAgICAgcmV0dXJuIHI7XG4gICAgICAgICAgICAgICAgfVxuICAgICAgICAgICAgfSk7XG4gICAgICAgICAgICB2YXIgXyA9IHUoNDUxMiksIGkgPSAhMDtcbiAgICAgICAgfSxcbiAgICAgICAgNzEzOTogZnVuY3Rpb24obiwgdCwgdSkge1xuICAgICAgICAgICAgKHdpbmRvdy5fX05FWFRfUCA9IHdpbmRvdy5fX05FWFRfUCB8fCBbXSkucHVzaChbXG4gICAgICAgICAgICAgICAgXCIvc3RhdGljXCIsXG4gICAgICAgICAgICAgICAgZnVuY3Rpb24oKSB7XG4gICAgICAgICAgICAgICAgICAgIHJldHVybiB1KDI5NDMpO1xuICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgIF0pO1xuICAgICAgICB9XG4gICAgfSxcbiAgICBmdW5jdGlvbihuKSB7XG4gICAgICAgIG4uTygwLCBbXG4gICAgICAgICAgICA3NzQsXG4gICAgICAgICAgICA4ODgsXG4gICAgICAgICAgICAxNzlcbiAgICAgICAgXSwgZnVuY3Rpb24oKSB7XG4gICAgICAgICAgICByZXR1cm4gbihuLnMgPSA3MTM5KTtcbiAgICAgICAgfSksIF9OX0UgPSBuLk8oKTtcbiAgICB9XG5dKTtcbiIKICBdLAogICJ2ZXJzaW9uIjogMwp9Cg==)

Reply:
`lookup_token` with `filter` was the cause.

[Visualized](https://evanw.github.io/source-map-visualization/#MTAxNQAoc2VsZi53ZWJwYWNrQ2h1bmtfTl9FID0gc2VsZi53ZWJwYWNrQ2h1bmtfTl9FIHx8IFtdKS5wdXNoKFsKICAgIFsKICAgICAgICAxNTgKICAgIF0sCiAgICB7CiAgICAgICAgMjk0MzogZnVuY3Rpb24obiwgdCwgdSkgewogICAgICAgICAgICAidXNlIHN0cmljdCI7CiAgICAgICAgICAgIHZhciByID0gZnVuY3Rpb24obikgewogICAgICAgICAgICAgICAgdmFyIHQgPSBuLmRhdGE7CiAgICAgICAgICAgICAgICByZXR1cm4gKDAsIF8uanN4KSgiZGl2IiwgewogICAgICAgICAgICAgICAgICAgIGNoaWxkcmVuOiB0LmZvbwogICAgICAgICAgICAgICAgfSk7CiAgICAgICAgICAgIH07CiAgICAgICAgICAgIHUucih0KSwgdS5kKHQsIHsKICAgICAgICAgICAgICAgIF9fTl9TU0c6IGZ1bmN0aW9uKCkgewogICAgICAgICAgICAgICAgICAgIHJldHVybiBpOwogICAgICAgICAgICAgICAgfSwKICAgICAgICAgICAgICAgIGRlZmF1bHQ6IGZ1bmN0aW9uKCkgewogICAgICAgICAgICAgICAgICAgIHJldHVybiByOwogICAgICAgICAgICAgICAgfQogICAgICAgICAgICB9KTsKICAgICAgICAgICAgdmFyIF8gPSB1KDQ1MTIpLCBpID0gITA7CiAgICAgICAgfSwKICAgICAgICA3MTM5OiBmdW5jdGlvbihuLCB0LCB1KSB7CiAgICAgICAgICAgICh3aW5kb3cuX19ORVhUX1AgPSB3aW5kb3cuX19ORVhUX1AgfHwgW10pLnB1c2goWwogICAgICAgICAgICAgICAgIi9zdGF0aWMiLAogICAgICAgICAgICAgICAgZnVuY3Rpb24oKSB7CiAgICAgICAgICAgICAgICAgICAgcmV0dXJuIHUoMjk0Myk7CiAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgIF0pOwogICAgICAgIH0KICAgIH0sCiAgICBmdW5jdGlvbihuKSB7CiAgICAgICAgbi5PKDAsIFsKICAgICAgICAgICAgNzc0LAogICAgICAgICAgICA4ODgsCiAgICAgICAgICAgIDE3OQogICAgICAgIF0sIGZ1bmN0aW9uKCkgewogICAgICAgICAgICByZXR1cm4gbihuLnMgPSA3MTM5KTsKICAgICAgICB9KSwgX05fRSA9IG4uTygpOwogICAgfQpdKTsKMTY1MgB7CiAgIm1hcHBpbmdzIjogIkFBQUNBLENBQUFBLEtBQUtDLG1CQUFtQkQsMkJBQTJCLE1BQU07SUFDdEQ7UUFDSTtLQUNIO0lBQ0Q7Y0FDVSxZQUFZSTtZQUNkO1lBQ0EsSUFBSUUsYUFBYUg7Z0JBQ2IsSUFBSUM7Z0JBQ0oseUJBQXlCO29CQUNyQk0sVUFBVU4sS0FBSztnQkFDbkI7WUFDSjtZQUNBQzs7OztnQkFJSVU7Ozs7WUFJSjs7MEJBRWNYOzs7O29CQUlOO2dCQUNKOzs7O2FBSUhEIiwKICAibmFtZXMiOiBbCiAgICAic2VsZiIsCiAgICAid2VicGFja0NodW5rX05fRSIsCiAgICAicHVzaCIsCiAgICAibiIsCiAgICAidCIsCiAgICAidSIsCiAgICAiciIsCiAgICAiZGF0YSIsCiAgICAiXyIsCiAgICAianN4IiwKICAgICJjaGlsZHJlbiIsCiAgICAiZm9vIiwKICAgICJkIiwKICAgICJfX05fU1NHIiwKICAgICJpIiwKICAgICJkZWZhdWx0IiwKICAgICJ3aW5kb3ciLAogICAgIl9fTkVYVF9QIiwKICAgICJPIiwKICAgICJzIiwKICAgICJfTl9FIgogIF0sCiAgInNvdXJjZXMiOiBbCiAgICAiLi4vLi4vaW5wdXQvaW5kZXguanMiCiAgXSwKICAic291cmNlc0NvbnRlbnQiOiBbCiAgICAiKHNlbGYud2VicGFja0NodW5rX05fRSA9IHNlbGYud2VicGFja0NodW5rX05fRSB8fCBbXSkucHVzaChbXG4gICAgW1xuICAgICAgICAxNThcbiAgICBdLFxuICAgIHtcbiAgICAgICAgMjk0MzogZnVuY3Rpb24obiwgdCwgdSkge1xuICAgICAgICAgICAgXCJ1c2Ugc3RyaWN0XCI7XG4gICAgICAgICAgICB2YXIgciA9IGZ1bmN0aW9uKG4pIHtcbiAgICAgICAgICAgICAgICB2YXIgdCA9IG4uZGF0YTtcbiAgICAgICAgICAgICAgICByZXR1cm4gKDAsIF8uanN4KShcImRpdlwiLCB7XG4gICAgICAgICAgICAgICAgICAgIGNoaWxkcmVuOiB0LmZvb1xuICAgICAgICAgICAgICAgIH0pO1xuICAgICAgICAgICAgfTtcbiAgICAgICAgICAgIHUucih0KSwgdS5kKHQsIHtcbiAgICAgICAgICAgICAgICBfX05fU1NHOiBmdW5jdGlvbigpIHtcbiAgICAgICAgICAgICAgICAgICAgcmV0dXJuIGk7XG4gICAgICAgICAgICAgICAgfSxcbiAgICAgICAgICAgICAgICBkZWZhdWx0OiBmdW5jdGlvbigpIHtcbiAgICAgICAgICAgICAgICAgICAgcmV0dXJuIHI7XG4gICAgICAgICAgICAgICAgfVxuICAgICAgICAgICAgfSk7XG4gICAgICAgICAgICB2YXIgXyA9IHUoNDUxMiksIGkgPSAhMDtcbiAgICAgICAgfSxcbiAgICAgICAgNzEzOTogZnVuY3Rpb24obiwgdCwgdSkge1xuICAgICAgICAgICAgKHdpbmRvdy5fX05FWFRfUCA9IHdpbmRvdy5fX05FWFRfUCB8fCBbXSkucHVzaChbXG4gICAgICAgICAgICAgICAgXCIvc3RhdGljXCIsXG4gICAgICAgICAgICAgICAgZnVuY3Rpb24oKSB7XG4gICAgICAgICAgICAgICAgICAgIHJldHVybiB1KDI5NDMpO1xuICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgIF0pO1xuICAgICAgICB9XG4gICAgfSxcbiAgICBmdW5jdGlvbihuKSB7XG4gICAgICAgIG4uTygwLCBbXG4gICAgICAgICAgICA3NzQsXG4gICAgICAgICAgICA4ODgsXG4gICAgICAgICAgICAxNzlcbiAgICAgICAgXSwgZnVuY3Rpb24oKSB7XG4gICAgICAgICAgICByZXR1cm4gbihuLnMgPSA3MTM5KTtcbiAgICAgICAgfSksIF9OX0UgPSBuLk8oKTtcbiAgICB9XG5dKTtcbiIKICBdLAogICJ2ZXJzaW9uIjogMwp9Cg==)